### PR TITLE
Update to 0.7 and 1.0

### PR DESCRIPTION
**Pull request by timholy**

### CODE REVIEW

File: src/slicing.jl [hunk 1]
```diff
@@ -56,24 +56,24 @@ function roi(inds::Indices, dims::Dims{2})
...
     for (i, n) in enumerate(names)
-        if !(n ∈ axes)
+        if !(n ∈ axnames)
             ind = inds[i]
             push!(sigs, Signal(first(ind)))
             push!(axs, Axis{n}(ind))
         end
     end
-    Signal(zr), SliceData{dims[2] < dims[1]}((sigs...), (axs...))
+    Signal(zr), SliceData{dims[2] < dims[1]}((sigs...,), (axs...,))
```
Comment (Evizero) [inline src/slicing.jl]: is the trailing `,` here needed now?

**Comment by timholy** [comment]
Waits on https://github.com/JuliaLang/METADATA.jl/pull/17497 (which I had forgotten was justifiably blocked), but otherwise this is hopefully good to go.

File: src/slicing.jl (hunk 1 above)
Comment (timholy) [reply src/slicing.jl]: I think so, and in fact I believe this was added by Femtocleaner (which I ran locally).
```julia
julia> x = (1,2)
(1, 2)

julia> s = (x...)
┌ Warning: Deprecated syntax `(x...)` at REPL[2]:1.
│ Use `(x...,)` instead.
└ @ REPL[2]:1
(1, 2)
```

**Comment by timholy** [comment]
Seems blocked by https://github.com/JuliaIO/ImageMagick.jl/issues/133

**Comment by RalphAS** [comment]
If you want the Travis tests to run, you need to force use of a newer `libz.so`.  (Gtk.jl loads the system version which is too old on the Ubuntu used by Travis.) This can be done by evaluating `import ImageMagick` before running the `Pkg.test` function. (Maybe `Pkg.add("ImageMagick") first.)

**Comment by RalphAS** [comment]
Sorry for the misleading advice; I forgot that Pkg runs tests in a separate environment.  I think the `import ImageMagick` has to go at the top of `runtests.jl` (before `using ImageView` or `using Grk`).

**Comment by timholy** [comment]
It's also weird that QuartzImageIO got installed. I will look into this later, gotta run now.

**Comment by timholy** [comment]
So this was worth merging, but I worry that users of older Ubuntu versions will encounter the same problem and have to manually use the same workaround. Short of building the new version statically into ImageMagick, I'm not sure I see a fix. Do you?

**Comment by RalphAS** [comment]
This is an example of a larger problem in the foreign library business. I got a bit carried away and raised [an alarm about this](https://discourse.julialang.org/t/dll-problems-episode-2/12405) on Discourse, where Stefan K made it clear that the Pkg crew hopes to improve the situation.

In this instance, there has been talk of using BinaryBuilder for Gtk, which might fix it. (I'm skeptical of that project, given how deep the linking of Gtk goes into the system, but I'll be happy to be surprised.)

In the interim, it would be good to provoke more useful error messages. That's a nice challenge.

**Pull request merged**